v2/periphery/DeployProxyAndInstallPlugin.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-3-0-or-later
+pragma solidity >=0.8.13;
+
+import { IPRBProxy } from "@prb/proxy/interfaces/IPRBProxy.sol";
+import { IPRBProxyRegistry } from "@prb/proxy/interfaces/IPRBProxyRegistry.sol";
```

Comment:
These should be re-exported in `@sablier/v2-periphery` in the `src/types` directory.

Reply:
Resolved in the final commit